dmapiclient/base.py

```diff
@@ -76,6 +76,9 @@ def _delete_with_updated_by(self, url, data, user):
     def _request(self, method, url, data=None, params=None):
         if not self.enabled:
             return None
+        if self.base_url is None:
+            logger.info("{} has no URL configured".format(self.__class__.__name__))
```

Comment:
I'm not sure about this. This is an error condition in any actual environment, so why would we want to hide this?